contributing/labeling.md

```diff
@@ -0,0 +1,15 @@
+# Label Definitions
+
+* `Epic`: Big issue to be broken down into smaller tasks
+
+### Priority (`p.*`)
+* `p.Critical`: To be handled as soon as possible, usually a bug or security loophole
+* `p.High`: Issues corresponding to the upcoming `Milestone`
+* `p.Medium`: Issues corresponding to a future `Milestone`
+* `p.Low`: Good-to-have features not in the roadmap
+
+### Difficulty (`d.*`)
```

Comment:
imho `labels` should not be used to categorise things but allow selection based on use cases. If you have a label for everything, they end up being worthless.